var/spack/repos/builtin/packages/py-fortls/package.py

```diff
@@ -16,11 +16,16 @@ class PyFortls(PythonPackage):
 
     license("MIT")
 
+    version(
+        "3.1.0", sha256="e38f9f6af548f78151d54bdbb9884166f8d717f8e147ab1e2dbf06b985df2c6d"
+    )  # FIXME
```

Comment:
Actually, let's go with
```suggestion
    version("3.1.0", sha256="e38f9f6af548f78151d54bdbb9884166f8d717f8e147ab1e2dbf06b985df2c6d")
```